db/migrate/20200925160112_sanitize_request_item_types.rb

```diff
@@ -0,0 +1,10 @@
+class SanitizeRequestItemTypes < ActiveRecord::Migration[6.0]
+  def up
+    Request.find_each do |request|
+      request.request_items = request.request_items&.map do |item|
+        item.merge("item_id" => item["item_id"]&.to_i, "quantity" => item["quantity"]&.to_i)
+      end
+      request.save
+    end
```

Comment:
@xjunior this is good. However, I generally try to avoid using ActiveRecord models like `Request` in migrations as it builds a dependency on `Request`. This migration would not run if we in the future renamed or deleted the `Request` model.

 I don't foresee that happening so I think we are safe to keep this in as it is :)